src/views/transactions/force-exit/force-exit.view.tsx

```diff
@@ -0,0 +1,257 @@
+                poolTransactionsTask={poolTransactionsTask}
+                preferredCurrency={preferredCurrency}
+                fiatExchangeRates={
+                  fiatExchangeRatesTask.status === "successful" ||
+                  fiatExchangeRatesTask.status === "reloading"
+                    ? fiatExchangeRatesTask.data
+                    : {}
+                }
+                pendingDeposits={[]}
+                onLoadAccounts={onLoadAccounts}
+                onAccountClick={(account: Account) => onGoToBuildTransactionStep(account)}
+              />
+            );
+          }
+          case "build-transaction": {
+            return (
+              account && (
+                <TransactionForm
+                  transactionType={TxType.ForceExit}
+                  account={account}
+                  preferredCurrency={preferredCurrency}
+                  fiatExchangeRates={
+                    fiatExchangeRatesTask.status === "successful" ||
+                    fiatExchangeRatesTask.status === "reloading"
+                      ? fiatExchangeRatesTask.data
+                      : {}
+                  }
+                  tokensPriceTask={tokensPriceTask}
+                  // ToDo: To be removed START
+                  receiverAddress={undefined}
+                  feesTask={{ status: "successful", data: null }}
+                  accountBalanceTask={{ status: "pending" }}
+                  estimatedWithdrawFeeTask={{ status: "pending" }}
+                  estimatedDepositFeeTask={{ status: "pending" }}
+                  onLoadAccountBalance={() => ({})}
+                  onLoadEstimatedWithdrawFee={() => ({})}
+                  onLoadEstimatedDepositFee={() => ({})}
+                  onLoadFees={() => ({})}
+                  // ToDo: To be removed END
+                  onSubmit={onGoToTransactionOverviewStep}
+                  onGoToChooseAccountStep={onGoToChooseAccountStep}
+                />
+              )
+            );
+          }
+          case "review-transaction": {
+            return (
+              wallet &&
+              account &&
+              transaction && (
+                <TransactionOverview
+                  wallet={wallet}
+                  isTransactionBeingApproved={isTransactionBeingApproved}
+                  transaction={{
+                    type: TxType.ForceExit,
+                    amount: transaction.amount,
+                    account: account,
+                    onForceExit,
+                  }}
+                  preferredCurrency={preferredCurrency}
+                  fiatExchangeRates={
+                    fiatExchangeRatesTask.status === "successful" ||
+                    fiatExchangeRatesTask.status === "reloading"
+                      ? fiatExchangeRatesTask.data
+                      : {}
+                  }
+                />
+              )
+            );
+          }
+          default: {
+            return <></>;
```

Comment:
I don't think this is required, `step` should be `never` here.

Reply:
Right! I've fixed it in all the flows.